app/components/footer/footer.module.css

```diff
@@ -0,0 +1,20 @@
+.footer {
+	position: absolute;
+	position: absolute;
+	bottom: 0;
```

Comment:
Absolute position isn't necessary. I think you can replace the modified position by a top margin.